node/core/approval-voting/src/backend.rs

```diff
@@ -0,0 +1,289 @@
+use kvdb::{DBTransaction, KeyValueDB};
```

Comment:
please add a license header to every new file, sir

Reply:
See, this is my bad... I knew I had to do this.